src/pages/clients/statement/Statement.tsx

```diff
@@ -102,6 +114,11 @@ export function Statement() {
   const iframeRef = useRef<HTMLIFrameElement>(null);
   const [selectRange, setSelectRange] = useState<string>('last_7_days');
 
+  const [client, setClient] = useState<Client>();
+
+  const [shouldEmailStatement, setShouldEmailStatement] =
+    useState<boolean>(false);
```

Comment:
There's no need for this, you can just break the send action when sending and return early with toast.

Reply:
Yeah, agree with you. The logic has been changed!